modules/primo/src/Primo/Ting/ObjectMapper.php

```diff
@@ -176,10 +176,7 @@ public function mapLanguage() {
    *   The series description, or FALSE if it could not be determined.
    */
   public function mapSeriesDescription() {
-    // Series data contain both title of series and number for current document.
-    // We only want the series title so split and return first element.
-    $seriesData = explode(' ; ', $this->document->getSeriesData());
-    return array_shift($seriesData);
+    return $this->document->getSeriesData();
```

Comment:
If we do not have any special processing we should just pass the value from the document to the object directly in the `map` function.